### PR TITLE
<QnAMaker> GenerateAnswer new field "StrictFiltersCompoundOperationType"

### PR DESCRIPTION
**Pull request by vipeketi**
QueryDTO has new field for Joining StrictFilters, StrictFiltersOperationType can take values AND, OR. Default value is AND. Test has been updated for OR operation.
https://github.com/Azure/azure-rest-api-specs/pull/10132#issuecomment-658321655

### CODE REVIEW

File: sdk/cognitiveservices/Knowledge.QnAMaker/src/Generated/Models/StrictFiltersCompoundOperationType.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.Models
+{
+
+    /// <summary>
+    /// Defines values for StrictFiltersCompoundOperationType.
+    /// </summary>
+    public static class StrictFiltersCompoundOperationType
```
Comment (bingisbestest) [inline sdk/cognitiveservices/Knowledge.QnAMaker/src/Generated/Models/StrictFiltersCompoundOperationType.cs:17]: Shoulnt this be enum?

File: sdk/cognitiveservices/Knowledge.QnAMaker/tests/QnAMakerRuntimeTests.cs [hunk 2]
```diff
@@ -19,7 +19,12 @@ public void QnAMakerRuntimeGenerateAnswerTest()
                 HttpMockServer.Initialize(this.GetType(), "QnAMakerRuntimeGenerateAnswerTest");
 
                 var client = GetQnAMakerRuntimeClient(HttpMockServer.CreateInstance());
-                var answer = client.Runtime.GenerateAnswerAsync("8758c6af-fa29-4e03-a517-9c36927f558f", new QueryDTO(question: "good afternoon")).Result;
+                var queryDTO = new QueryDTO();
+                queryDTO.StrictFiltersCompoundOperationType = StrictFiltersCompoundOperationType.OR;
+                queryDTO.StrictFilters = new List<MetadataDTO>();
+                queryDTO.StrictFilters.Add(new MetadataDTO("question", "good afternoon"));
+                queryDTO.StrictFilters.Add(new MetadataDTO("question", "good morning"));
+                var answer = client.Runtime.GenerateAnswerAsync("8758c6af-fa29-4e03-a517-9c36927f558f", queryDTO).Result;
```
Comment (bingisbestest) [inline sdk/cognitiveservices/Knowledge.QnAMaker/tests/QnAMakerRuntimeTests.cs:27]: whr is data file for test?

**Review by ghost — COMMENTED** [review]
also bump version in csproj and changes file and add to change log

File: sdk/cognitiveservices/Knowledge.QnAMaker/tests/QnAMakerRuntimeTests.cs (hunk 2 above)
Comment (vipeketi) [reply sdk/cognitiveservices/Knowledge.QnAMaker/tests/QnAMakerRuntimeTests.cs:27]: QnAMakerRuntimeTests\QnAMakerRuntimeGenerateAnswerTest.json

**Review by AlexGhiondea — APPROVED** [review]

**Pull request merged**